editor/Storyboarding/EditorGeneratorContext.cs

```diff
@@ -51,10 +51,10 @@ public EditorGeneratorContext(Effect effect, string projectPath, string mapsetPa
             this.watcher = watcher;
         }
 
-        public override StoryboardLayer GetLayer(string identifier)
+        public override StoryboardLayer GetLayer(string identifier, OsbLayer osbLayer = OsbLayer.Background)
         {
             var layer = EditorLayers.Find(l => l.Identifier == identifier);
-            if (layer == null) EditorLayers.Add(layer = new EditorStoryboardLayer(identifier, effect));
+            if (layer == null) EditorLayers.Add(layer = new EditorStoryboardLayer(identifier, effect, osbLayer));
```

Comment:
I don't really like that the osbLayer only does something the first time you call this...